Project1/project1/models.py

```diff
@@ -0,0 +1,12 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+class Users(db.Model):
+    __tablename__ = "USERS"
```

Comment:
for authentication username and passwprd will be enough to store in db

Reply:
time stamp is missing

Reply:
removed birthday and address column from database, added created_at timestamp column